src/main/java/org/rsa/command/v2/SubcommandObjectV2.java

```diff
@@ -0,0 +1,16 @@
+package org.rsa.command.v2;
+
+import net.dv8tion.jda.api.events.interaction.command.CommandAutoCompleteInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData;
+
+public abstract class SubcommandObjectV2 extends SubcommandData {
+
+    public SubcommandObjectV2(String name, String description) {
+        super(name, description);
+    }
+
+    public void processAutoCompleteInteraction(EventEntities<CommandAutoCompleteInteractionEvent> entities) {}
+
+    public void processSlashCommandInteraction(EventEntities<SlashCommandInteractionEvent> entities) {}
```

Comment:
These should have default responses to the events they are processing in the event a path where implementation is not intended but implementation is received.